### ISSUE TITLE
Failed to Load Plugin: Colorpicker, image from URL - issue with plugins

### ISSUE THREAD

**Issue by themak1985** [veda-consulting/uk.co.vedaconsulting.mosaico#598]
Very weird, I updated one of my site to 3.1 and it worked just fine, another site that had mosaico extras and tweaks has lots of problems, i got the following errors:

`Failed to load plugin: colorpicker from url https://example.org/wp-content/uploads/civicrm/ext/uk.co.vedaconsulting.mosaico/packages/mosaico/dist/rs/plugins/colorpicker/plugin.min.js
`
I was able to fix this by disabling mosaico tweaks...

`Failed to load plugin: image from url https://pandemicresponse.columbia.edu/wp-content/uploads/civicrm/ext/uk.co.vedaconsulting.mosaico/packages/mosaico/dist/rs/plugins/image/plugin.min.js`

Now this became an issue. but it was fixed by disabling mosaico extras.

Lastly on this new deployment, all the new styling features and blocks are missing. I think something with tweaks and extras blocked the 3.1 upgrade from being successful.

Steps taken to resolve: (all unsuccessful)
uninstalled and deleted tweaks and extras
Uninstalled and deleted mosaico
reinstalled mosaico using civicrm ui
Did not work - still looks like 2.12, eventhough upgraded to 3.1 - uninstalled mosaico again
Installed 2.12
Everything works
Upgraded to 3.1
Still operates like it is 2.12, but now social icons broken

Update, after sending an email yesterday - where mosaico was still acting like 2.12, today, everything seems to be working fine. Not sure what fixed it, but now its all fixed.

Wordpress, Civi 5.61.4

**Comment by mattwire** [comment]
@themak1985 I think that both tweaks and extras will need changes to be compatible - with 3.0 lots of library paths changed.

**Comment by themak1985** [comment]
Thanks @mattwire - I am trying to uninstall and reinstall mosaico like it is a fresh install and that is not working - any suggestions? I updated issue above with all the steps I have taken.

**Comment by yorkshirerose** [comment]
I've had the same issue:
Drupal 10.0.9 CiviCRM 5.62.1
New site install, installed 3.1 and then installed Mosaico Tweaks. 

Saw the same error:
`Failed to load plugin: colorpicker from url https://example.org/wp-content/uploads/civicrm/ext/uk.co.vedaconsulting.mosaico/packages/mosaico/dist/rs/plugins/colorpicker/plugin.min.js `

It looks like 2.12 (specfically the image uploads placeholders, which look slightly different in 3+) but my social media icons are not broken.

Disabled Mosaico Tweaks, the error has gone but it is behaving

**Comment by mattwire** [comment]
Yes, to confirm mosaico tweaks is NOT compatible with mosaico 3.x until someone funds the necessary work to make it compatible.
